bot/utils/messages.py

```diff
@@ -7,13 +7,15 @@
 from discord.abc import Snowflake
 from discord.errors import HTTPException
 
+from bot.constants import Emojis
+
 MAX_SIZE = 1024 * 1024 * 8  # 8 Mebibytes
 
 
 async def wait_for_deletion(
     message: Message,
     user_ids: Sequence[Snowflake],
-    deletion_emojis: Sequence[str] = ("❌",),
+    deletion_emojis: Sequence[str]=(Emojis.cross_mark,),
```

Comment:
`E252 missing whitespace around parameter equals`

```suggestion
    deletion_emojis: Sequence[str] = (Emojis.cross_mark,),
```